src/styles/util.scss

```diff
@@ -10,6 +10,15 @@
   @include layout.row;
 }
 
+.u-layout-row--align-center,
```

Comment:
Are these `u-*` rules going to be used later somewhere?

Reply:
If they're not used in this set of changes, they may have gotten split up from their use when I reordered some commits. They're used somewhere!

After I land the queue of changes I've got, I'll grep to make sure all of the utility classes are used. The `u-*` prefix helps with that!